lua/tint.lua

```diff
@@ -1,121 +1,142 @@
+local colors = require("tint.colors")
+
 local tint = {
   config = {
-    amt = -40,
+    tint = -40,
     saturation = 0.7,
-    bg = false,
-    ignore = {},
-    ignorefunc = nil,
+    transforms = nil,
+    tint_background_colors = false,
+    highlight_ignore_patterns = {},
+    window_ignore_function = nil,
+  },
+  transforms = {
+    SATURATE_TINT = "saturate_tint",
   },
 }
 
-local __ = {}
-
-local function get_hex(val)
-  -- Convert decimal to hex
-  return string.format("%06x", val)
-end
+-- Private "namespace" for functions, etc. that might not be defined before they are used
+local __ = {
+  transforms = {
+    [tint.transforms.SATURATE_TINT] = {
+      colors.saturate(tint.config.saturation),
+      colors.tint(tint.config.tint),
```

Comment:
Wouldn't this need to be evaluated after the `.setup()` function has been called. By just looking at this it seems like it'll only ever use the default values, but I might be missing something?

Reply:
Yes, I just changed it to be a function that gets called in `get_transforms` to fix this issue when testing it locally.